client/sites-dashboard-v2/sites-dataviews/dataviews-fields/site-field.tsx

```diff
@@ -97,23 +84,25 @@ const SiteField = ( { site, openSitePreviewPane }: Props ) => {
 					`
 				) }
 				leading={
-					<ListTileLeading title={ title }>
-						<SiteItemThumbnail
-							className="sites-site-thumbnail"
-							displayMode="list"
-							showPlaceholder={ false }
-							site={ site }
-						/>
-						<SiteFavicon
-							className="sites-site-favicon"
-							blogId={ site.ID }
-							isDotcomSite={ site.is_wpcom_atomic }
-						/>
-					</ListTileLeading>
+					<Button className="sites-dataviews__preview-trigger" onClick={ onSiteClick } borderless>
```

Comment:
We could potentially remove the need for a button here entirely in a follow-up PR since the entire row is clickable. The row-click mechanism could fetch the site instead of relying on this button.